src/main/java/gmbs/model/lotto/number/LottoTicket.java

```diff
@@ -0,0 +1,72 @@
+package gmbs.model.lotto.number;
+
+import gmbs.model.lotto.number.vo.LottoNumber;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public final class LottoTicket {
+
+    private static final int LOTTO_SIZE = 6;
+
+    private final List<LottoNumber> values;
+
+    private LottoTicket(final List<LottoNumber> lottoNumbers) {
+        this.values = lottoNumbers;
+    }
+
+    public static LottoTicket from(final List<Integer> numbers) {
+        validateSize(numbers);
+        validateDuplicateNumber(numbers);
+        return new LottoTicket(
+                numbers.stream()
+                        .map(LottoNumber::getInstance)
+                        .sorted()
+                        .collect(Collectors.toUnmodifiableList()));
+    }
+
+    private static void validateSize(final List<Integer> numbers) {
+        if (numbers.size() != LOTTO_SIZE) {
+            throw new IllegalArgumentException("[ERROR] 로또 번호는 6개여야 합니다");
+        }
+    }
+
+    private static void validateDuplicateNumber(final List<Integer> numbers) {
+        if (numbers.stream().distinct().count() != LOTTO_SIZE) {
+            throw new IllegalArgumentException("[ERROR] 로또 번호에 중복이 있습니다");
+        }
+    }
+
+    public int calculateMatchCountByWinNumbers(final LottoTicket winNumbers) {
+        final List<LottoNumber> tempLottoNumbers = new ArrayList<>(values);
+        tempLottoNumbers.retainAll(winNumbers.getLottoNumbers());
+        return tempLottoNumbers.size();
+    }
+
+    public boolean hasBonus(final LottoNumber bonusNumber) {
+        return values.contains(bonusNumber);
+    }
+
+    public List<LottoNumber> getLottoNumbers() {
+        return Collections.unmodifiableList(values);
+    }
+
+    public List<Integer> getLottoNumbersValues() {
```

Comment:
```suggestion
    public List<Integer> getLottoNumberValues() {
```
getLottoNumberValues라는 네이밍이 더 좋은 것 같은데 어떻게 생각하시나요~?

Reply:
저도 두개 중에 고민하긴 했는데, 아무래도 LottoNumber 들의 value 값들 이라서  저렇게 네이밍 했습니다 ㅎㅎ

Reply:
제 생각에는 LottoNumber 내부에 있는 value 값 -> LottoNumberValue
LottoNumberValue 들이 여러개다  -> LottoNumberValues 이렇게 생각했거든요!
그리고 List<Integer>고 해당 객체에서 Integer로 다 바꿔서 나오니까요!

사실 크게 중요하지 않습니다~ㅋㅋㅋ